src/System.Linq.Expressions/tests/New/NewTests.cs

```diff
@@ -466,16 +466,46 @@ public static IEnumerable<object[]> Type_InvalidType_TestData()
             yield return new object[] { typeof(int).MakeByRefType() };
             yield return new object[] { typeof(StaticCtor) };
             yield return new object[] { typeof(ClassWithNoDefaultCtor) };
+            yield return new object[] { typeof(int).MakePointerType() };
```

Comment:
Yeah I think they're equivalent - I've put `makepointertype` Instead of `int*` in cleanups before